libraries/chain/webassembly/crypto.cpp

```diff
@@ -117,51 +117,26 @@ namespace eosio { namespace chain { namespace webassembly {
    }
 
    int32_t interface::alt_bn128_add(span<const char> op1, span<const char> op2, span<char> result ) const {
-      bytes bop1(op1.data(), op1.data() + op1.size());
-      bytes bop2(op2.data(), op2.data() + op2.size());
-
-      auto maybe_err = fc::alt_bn128_add(bop1, bop2);
-      if(std::holds_alternative<fc::alt_bn128_error>(maybe_err)) {
+      if (op1.size() < 64 ||  op2.size() < 64 ||  result.size() < 64 ||
```

Comment:
I think it's only `result` that should be  changed to `< 64`. In the previous impl `op1` and `op2` are copied in to `bytes` (vectors) that are the same size as the `op1` and `op2` inputs before being fed to `alt_bn128_add()`:
https://github.com/AntelopeIO/leap/blob/22daed5030ba5bc9bac81360c8c6ddd58b0d434f/libraries/chain/webassembly/crypto.cpp#L120-L123
That is then,
https://github.com/AntelopeIO/leap/blob/22daed5030ba5bc9bac81360c8c6ddd58b0d434f/libraries/libfc/src/crypto/alt_bn128.cpp#L142-L153
and `decode_g1_element()` does,
https://github.com/AntelopeIO/leap/blob/22daed5030ba5bc9bac81360c8c6ddd58b0d434f/libraries/libfc/src/crypto/alt_bn128.cpp#L42-L45